src/go/k8s/api/redpanda/v1alpha1/redpanda_clusterspec_types.go

```diff
@@ -164,6 +166,17 @@ type ConsoleCreateObj struct {
 	Create bool `json:"create"`
 }
 
+type RedpandaConnectors struct {
+	Enabled *bool `json:"enabled,omitempty"`
+
+	Deployment *ConnectorsCreateObj `json:"deployment,omitempty"`
+	Test       *ConnectorsCreateObj `json:"test,omitempty"`
+}
+
+type ConnectorsCreateObj struct {
+	Create *bool `json:"enabled,omitempty"`
+}
```

Comment:
This should be only part of the configuration. It only reflects [Redpanda values](https://github.com/redpanda-data/helm-charts/blob/cc2b2ed1ceb25fea45790fe2c298f29fd785cee1/charts/redpanda/values.yaml#L129-L138) where user could define [Connectors values](https://github.com/redpanda-data/helm-charts/blob/cc2b2ed1ceb25fea45790fe2c298f29fd785cee1/charts/connectors/values.yaml) in standard helm deployment.

Reply:
Yes. Consider this the first part. For now I want to unblock users so they can deploy connectors.